docs/Project-Configuration/Merge-Queue.md

```diff
@@ -22,8 +22,7 @@ merge, like you would have to without the queue.
 1. From <https://spruce.mongodb.com/>, from the More drop down, select Project Settings.
 2. Select your project from the project dropdown.
 3. From the GitHub & Commit Queue section, set the Commit Queue to Enabled.
```

Comment:
just GitHub (the `& Commit Queue` part is removed).

Commit Queue -> Merge Queue